src/haddock/modules/topology/topocg/aa2cg.py

```diff
@@ -24,14 +24,15 @@
 from Bio.PDB import Entity
 from Bio.PDB import PDBIO
 from Bio.PDB import PDBParser
-from Bio.PDB.DSSP import DSSP
 from Bio.PDB.StructureBuilder import StructureBuilder
 
 from haddock.core.exceptions import ModuleError
 from haddock.modules.topology.topocg.helper import *
 
 warnings.filterwarnings("ignore")
 
+CRYST_LINE = "CRYST1 \n"
```

Comment:
Use `os.linesep` instead of `\n` to avoid problems in the future and to keep in line with the rest of the code

```python
CRYST_LINE = "CRYST1 " + os.linesep
```